app/resources/numismatic_issues/numismatic_issue_change_set.rb

```diff
@@ -69,47 +69,55 @@ class NumismaticIssueChangeSet < ChangeSet
   validates :visibility, presence: true
 
   def primary_terms
-    [
-      :rights_statement,
-      :rights_note,
-      :member_of_collection_ids,
-      :artist,
-      :color,
-      :date_object,
-      :date_range,
-      :denomination,
-      :department,
-      :description,
-      :edge,
-      :era,
-      :geographic_origin,
-      :issue_number,
-      :master,
-      :metal,
-      :note,
-      :object_type,
-      :obverse_attributes,
-      :obverse_figure,
-      :obverse_figure_description,
-      :obverse_figure_relationship,
-      :obverse_legend,
-      :obverse_orientation,
-      :obverse_part,
-      :obverse_symbol,
-      :place,
-      :reverse_attributes,
-      :reverse_figure,
-      :reverse_figure_description,
-      :reverse_figure_relationship,
-      :reverse_legend,
-      :reverse_orientation,
-      :reverse_part,
-      :reverse_symbol,
-      :ruler,
-      :series,
-      :shape,
-      :subject,
-      :workshop
-    ]
+    {
+      "" => [
+        :issue_number,
```

Comment:
one question. why is this key empty?

Reply:
To let you make a "default" section without a header: https://github.com/pulibrary/figgy/pull/2182/files#diff-f2e3b8d6bd878f3f8c406fc12a48ef01R8